user_domain.endpoint.http/src/external_task/external_task_router.ts

```diff
@@ -7,12 +9,16 @@ import {restSettings} from '@atlas-engine/user_domain.contracts';
 
 import {ExternalTaskController} from './external_task_controller';
 
+@injectable()
 export class ExternalTaskRouter extends BaseRouter {
 
   private identityService: IIdentityService;
   private externalTaskController: ExternalTaskController;
 
-  constructor(identityService: IIdentityService, externalTaskController: ExternalTaskController) {
+  constructor(
+  @inject('IdentityService') identityService: IIdentityService,
```

Comment:
Wie kommts, dass der Linter hier nicht angeschlagen hat, bzw. das automatisch gefixt wurde?

```suggestion
    @inject('IdentityService') identityService: IIdentityService,
```

Reply:
Das wurde so automatisch "gefixt". Ich kann das auch nicht ändern ohne dass er das zurück ändert.

Reply:
Na super. 😖